src/main/java/de/rub/nds/anvilcore/teststate/AnvilTestRun.java

```diff
@@ -198,14 +342,20 @@ public void finish() {
             // only determine result automatically if it has not been set before
             result = resolveFinalResult();
         }
-        if (result == TestResult.DISABLED && getDisabledReason() != null) {
-            LOGGER.info("{} is disable because {}", getTestMethodName(), getDisabledReason());
-        }
         scoreContainer.updateForResult(result);
-        AnvilContext.getInstance()
-                .addTestResult(result, testClass.getName() + "." + testMethod.getName());
+        AnvilContext.getInstance().addTestResult(result, this);
         AnvilContext.getInstance().getMapper().saveTestRunToPath(this);
         AnvilContext.getInstance().testFinished(uniqueId);
+
+        // log test run summary
+        if (result == TestResult.DISABLED) {
+            LOGGER.info(
+                    "{} is disabled because {}",
+                    getTestMethodName() != null ? getTestMethodName() : "undefined",
+                    disabledReason != null ? disabledReason : "undefined");
+        } else {
+            logTestRun();
```

Comment:
We should limit these live-logs to tests that are not succeeded / conceptually succeeded.

Reply:
I adressed this issue in commit [61a8227](https://github.com/tls-attacker/Anvil-Core/pull/31/commits/61a82271fce73664ec16062134243728127e58f2)